install.sh

```diff
@@ -21,6 +21,31 @@ then
   eval "export PATH=$PATH:$DEST_DIR"
 fi
 
+if [ "$1" != --no-override ]
+then
+echo "Overriding existing git push, pull, and fetch commands to use secure versions"
```

Comment:
the contents of this if block ought to be indented. Also, we may want to add logic that scans the whole list of arguments for options, like we did for merge-pr et al., as we will likely be adding other option flags to the install script.

Reply:
Do you think we should also add a help flag with some usage instructions now that there are installation options, or is that silly overkill because presumably people read the the readme?